python/Ganga/Runtime/GPIexport.py

```diff
@@ -18,6 +18,31 @@
 from Ganga.GPIDev.Base.Objects import GangaObject
 from inspect import isclass
 
+def _addToInterface(interface, name, _object):
+
+    if isType(_object, GangaObject):
+        setattr(interface, name, addProxy(_object))
+    elif isclass(_object) and issubclass(_object, GangaObject):
+        setattr(interface, name, getProxyClass(_object))
+    else:
+        setattr(interface, name, _object)
+
+def exportToInterface(myInterface, name, _object, doc_section, docstring=None):
```

Comment:
I'm happy to keep the code as you've written it. About problems from people using `import Ganga.GPI` inside the codebase, those are things that must be fixed as they are violating the proxy layer. They should be accessed through a global variable or through the plugin system.

About the extra argument to this function: My reason for wanting to remove it (in the future) is that every single function, argument, class or module that you add increases the maintenance cost of the system. We must be thoughtful and careful about what we add and how it works. This is a classic example of technical debt.